scripts/bump.sh

```diff
@@ -59,20 +59,37 @@ echo "::set-output name=version::$NEXT_EXTENSION_VERSION"
 jq ".version = \"$NEXT_EXTENSION_VERSION\" | \
     .prisma.version = \"$SHA\" | \
     .dependencies[\"@prisma/get-platform\"] = \"$NPM_VERSION\"" \
-    ./../language-server/package.json > ./../language-server/package.json.bk
+    ./packages/language-server/package.json > ./packages/language-server/package.json.bk
 
-mv ./../language-server/package.json.bk ./../language-server/package.json
-mv ./package.json.bk ./package.json
+jq ".dependencies[\"@prisma/language-server\"] = \"$NEXT_EXTENSION_VERSION\"" \
+    ./packages/vscode/package.json > ./packages/vscode/package.json.bk
 
-npm install
+mv ./packages/language-server/package.json.bk ./packages/language-server/package.json
+mv ./packages/vscode/package.json.bk ./packages/vscode/package.json
 
 (
-cd ./../language-server
+cd ./packages/language-server
 npm install
+npm run build
 )
 
+if [ -z "$NODE_AUTH_TOKEN" ]; then
+    echo "\$NODE_AUTH_TOKEN is empty. Please set the value of $NODE_AUTH_TOKEN"
+elif [ -n "$NODE_AUTH_TOKEN" ]; then
+    if [ "$PRODUCTION" = "1" ]; then
+        echo "Publishing language-server"
+        cd ./packages/language-server && ./node_modules/.bin/npm publish && cd ../..
+    else
+        echo "Printing the command because PRODUCTION is not set"
+        echo "cd ./packages/language-server && ./node_modules/.bin/npm publish && cd ../.."
+    fi
+fi
```

Comment:
Found this out when making the patch locally. Publish should happen in `publish.sh` before the extension. `bump.sh` should only bump `package.json` and other files locally. 

`bump.sh` shouldn't have a side-effect like publishing itself.

Reply:
This is not possible though as npm install will fail in the client if the lsp is not published yet but the dependency is bumped.

Reply:
I can split the bump and publish script up though to make it more clear what happens there.

Reply:
The npm installs can also be moved to `publish.sh` in that case.

Reply:
yes, makes sense. Added that in the last commit 👍